content/techniques/validation.md

```diff
@@ -121,6 +121,8 @@ create(@Body() createUserDto: CreateUserDto) {
 
 > info **Hint** Since TypeScript does not store metadata about **generics or interfaces**, when you use them in your DTOs, `ValidationPipe` may not be able to properly validate incoming data.  For this reason, consider using concrete classes in your DTOs.
 
+> info **Hint** When importing your DTOs, you can't use a type-only import as that would be erased at Runtime, i.e. remember to `import { CreateUserDto }` instead of `import type { CreateUserDto }`.
```

Comment:
```suggestion
> info **Hint** When importing your DTOs, you can't use a type-only import as that would be erased at runtime, i.e. remember to `import {{ '{' }} CreateUserDto {{ '}' }}` instead of `import type {{ '{' }} CreateUserDto {{ '}' }}`.
```